proto/substrait/algebra.proto

```diff
@@ -131,14 +131,42 @@ message ReadRel {
       message ArrowReadOptions {}
       message OrcReadOptions {}
       message DwrfReadOptions {}
+      message DelimeterSeparatedTextReadOptions {
+        // Delimiter separated files may be compressed.  The reader should
+        // autodetect this and decompress as needed.
+
+        // The character(s) used to separate fields.  Common values are comma,
+        // tab, and pipe.  Multiple characters are allowed.
+        string field_delimiter = 1;
+        // The maximum number of bytes to read from a single line.  If a line
+        // exceeds this limit the resulting behavior is undefined.
+        uint64 max_line_size = 2;
+        // The character(s) used to quote strings.  Common values are single
+        // and double quotation marks.
+        string quote = 3;
+        // If true, consume the first row as names of the columns.  These names
+        // are not used elsewhere in the plan.
+        uint64 header = 4;
+        // The character(s) used to escape characters in strings.  Backslash is
+        // the most common value.
+        string escape = 5;
```

Comment:
Saying this can be more than character is interesting. Where have you seen multiple characters used? High performance readers are often hand tuned for this kind of thing so I'm wondering if we really need multiple characters.